app/screens/settings-screen/settings-screen.tsx

```diff
@@ -141,13 +144,38 @@ export const SettingsScreen: React.FC = () => {
       enabled: !isAuthed,
       greyed: isAuthed,
     },
+    {
+      category: LL.GaloyAddressScreen.yourAddress({ bankName: "BBW" }),
+      icon: "person",
+      id: "username",
+      subTitleDefaultValue: LL.SettingsScreen.tapUserName(),
+      subTitleText: lightningAddress,
+      action: () => {
+        if (!lightningAddress) {
+          navigation.navigate("addressScreen")
+          return
+        }
+        Clipboard.setString(lightningAddress)
+        toastShow({
+          message: (translations) =>
+            translations.GaloyAddressScreen.copiedAddressToClipboard({
+              bankName,
+            }),
+          type: "success",
+          currentTranslation: LL,
+        })
+      },
+      enabled: isAuthed,
+      greyed: !isAuthed,
+      hidden: !isAuthed,
```

Comment:
```suggestion
```

Reply:
I don't think we should have this hidden?

Reply:
fixed